coordinator/src/main/scala/filodb.coordinator/TenantIngestionMetering.scala

```diff
@@ -65,7 +65,7 @@ case class TenantIngestionMetering(settings: FilodbSettings,
     dsIterProducer().foreach { dsRef =>
       val fut = Client.asyncAsk(
         coordActorProducer(),
-        LogicalPlan2Query(dsRef, TsCardinalities(prefix, numGroupByFields)),
+        LogicalPlan2Query(dsRef, TsCardinalities(prefix, numGroupByFields, 2, Seq("longtime-prometheus"))),
```

Comment:
@alextheimer @amolnayak311 Do you know if we have longtime-prometheus as the default dataset for single partition planner for all partitions ?